plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -273,13 +284,18 @@ export class PgInstrumentation extends InstrumentationBase<PgInstrumentationConf
   private recordOperationDuration(attributes: Attributes, startTime: HrTime) {
     const metricsAttributes: Attributes = {};
     const keysToCopy = [
-      SEMATTRS_DB_SYSTEM,
       ATTR_DB_NAMESPACE,
       ATTR_ERROR_TYPE,
       ATTR_SERVER_PORT,
       ATTR_SERVER_ADDRESS,
       ATTR_DB_OPERATION_NAME,
     ];
+    if (this._semconvStability & SemconvStability.OLD) {
+      keysToCopy.push(SEMATTRS_DB_SYSTEM);
```

Comment:
nit: could change `SEMATTRS_DB_SYSTEM` to `ATTR_DB_SYSTEM` (importing it from './semconv'), or could do that in a separate change.

Same for the other `SEMATTRS_...` usages in this package.

Reply:
done